docs/04-reference/winglang-spec.md

```diff
@@ -406,7 +406,7 @@ Variables can be reassigned to by adding the `var` modifier:
 
 ```ts
 // wing
-let var sum = 0;
+var sum = 0;
```

Comment:
nope, our syntax is `let var` :-)

Reply:
oh was not aware of it, tried to look for reference in the doc, but no luck. probably I missed it somewhere, any pointer?

Reply:
`Let let be let. (Elad B. 2022)` I figured